bytecode/instr.py

```diff
@@ -306,9 +316,15 @@ def stack_effect(self, jump=None):
         # then we should pass that through, and perhaps remove some of the
         # overrides that are set up in _init_stack_effects()
 
-        # All opcodes whose arguments are not represented by integers have
-        # a stack_effect indepent of their argument.
-        arg = (self._arg if isinstance(self._arg, int) else
+        # Each of following opcodes has a stack_effect indepent of its
```

Comment:
```suggestion
        # Each of the following opcodes has a stack_effect independent of its
```

Reply:
I do want to accept all of your suggestions, but it seems that I have to make lots of commits  because each of suggestion application performs a commit.

Did I break PEP8? Or just my word is verbose...

Reply:
Do not worry about the commits I will most likely squash them when merging.

Reply:
@MatthieuDartiailh Oh, I've just made a new commit from the local repo... My bad.